docs/src/dedicated-gen-3/overview/_index.md

```diff
@@ -36,15 +37,15 @@ This means that you can edit your configuration yourself
 - A minimum of three virtual machine cluster is associated with your default (production) environment, and also optionally with a staging environment.
```

Comment:
I put a temporary structure into place for the section for now, but it will no doubt change again when we get to reworking the actual content on each page. Structure & content in the DG2 section will need to be rethought as well in a later PR.